activerecord/lib/active_record/counter_cache.rb

```diff
@@ -18,11 +18,11 @@ module ClassMethods
       #
       # ==== Examples
       #
-      #   # For Post with id #1 records reset the comments_count
+      #   # For the Post with id #1 record, reset the comments_count
```

Comment:
I think we can remove the `record`, based on reading seems clear that "record" is implied.